### PR TITLE
Datetime function is moved to commons

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: root <root@dhcp42-75.lab.eng.blr.redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n please review

**Review by shtripat — COMMENTED** [review]
LGTM.
Have you verified the import cluster flow post this once?

File: tendrl/ceph_integration/util.py [hunk 1]
```diff
@@ -1,16 +1,8 @@
-import datetime
 from pytz import utc
```
Comment (r0h4n) [inline tendrl/ceph_integration/util.py]: remove pytz import and remove pytz from requirements

cc @shtripat

**Review by r0h4n — CHANGES_REQUESTED** [review]

**Comment by shtripat** [comment]
@GowthamShanmugam where is commons PR for this?

**Comment by r0h4n** [comment]
@shtripat  https://github.com/Tendrl/commons/pull/134

**Comment by GowthamShanmugam** [comment]
@r0h4n changes done

**Comment by GowthamShanmugam** [comment]
@shtripat still not verified with import cluster i will try

**Comment by GowthamShanmugam** [comment]
@r0h4n all changes are done

**Comment by GowthamShanmugam** [comment]
@shtripat i have verified with import cluster it is working fine.. for changes in common i updated with logging PR

**Pull request closed without merging**